autogpt/agent/agent.py

```diff
@@ -106,7 +106,7 @@ def start_interaction_loop(self):
                     f"ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}",
                 )
                 print(
-                    "Enter 'y' to authorise command, 'y -N' to run N continuous "
+                    "Enter 'y' or Enter directly to authorise command, 'y -N' to run N continuous "
```

Comment:
```suggestion
                    "Enter 'y' or Enter to authorize command, 'y -N' to run N continuous "
```

Sorry about second (!) nitpick. Feel free to ignore it. Just offering my impression on this.

I think `directly` has no meaning here. At least I fail to understand the need. And authorize with `z` is more common. I get `s` is British spelling. `z` is american. I would go with it because it is mainstream. But I am biased.

I wasn't even sure if I should be commenting on this. But e-mail notified me so I had a second look.

Reply:
Shorter is better.

<img src="https://media1.giphy.com/media/NEvPzZ8bd1V4Y/giphy.gif"/>